packages/cli/README.md

```diff
@@ -34,6 +36,16 @@ glaze COMMAND
 - [`glaze model:inspect NAME`](#glaze-modelinspect-name)
 - [`glaze model:list`](#glaze-modellist)
 - [`glaze model:publish NAME [OUTPUT]`](#glaze-modelpublish-name-output)
+- [`glaze pin:add STREAMID`](#glaze-pinadd-streamid)
+- [`glaze pin:ls [STREAMID]`](#glaze-pinls-streamid)
+- [`glaze pin:rm STREAMID`](#glaze-pinrm-streamid)
+- [`glaze stream:commits STREAMID`](#glaze-streamcommits-streamid)
+- [`glaze stream:state STREAMID`](#glaze-streamstate-streamid)
+- [`glaze tile:create CONTENT`](#glaze-tilecreate-content)
+- [`glaze tile:deterministic METADATA`](#glaze-tiledeterministic-metadata)
+- [`glaze tile:show STREAMID`](#glaze-tileshow-streamid)
+- [`glaze tile:update STREAMID CONTENT`](#glaze-tileupdate-streamid-content)
+- [`glaze tile:watch STREAMID`](#glaze-tilewatch-streamid)
```

Comment:
this feature does not work today in the Ceramic CLI.  Does it work here?

Reply:
This feature hasn't been tested, I only wanted to bring it over if it was working. If it isn't working then I'm going to remove it.